components/esp_modem/test/host_test/env.sh

```diff
@@ -1,10 +1,20 @@
 #!/bin/bash
 
-lwip=$1
-lwip_uri=$2
-lwip_contrib=$3
-
-wget --no-verbose ${lwip_uri}/${lwip}.zip
-unzip -oq ${lwip}.zip
-wget --no-verbose ${lwip_uri}/${lwip_contrib}.zip
-unzip -oq ${lwip_contrib}.zip
+idf_version=$1
+component=$2
+
+if [ "$idf_version" == "release-v4.3" ] && [ "$component" == "esp_modem" ]; then
```

Comment:
probably need something like... ?
```suggestion
if [[ "$idf_version" == "release-v4.3" ]] && [[ "$component" == "esp_modem" ]]; then
```